hazelcast/src/hazelcast/client/compact.cpp

```diff
@@ -1500,12 +1500,12 @@ init_fp_table()
     return FP_TABLE;
 }
 
-static std::array<uint64_t, 256> FP_TABLE = init_fp_table();
 constexpr uint64_t rabin_finger_print::INIT;
 
 uint64_t
 rabin_finger_print::fingerprint64(uint64_t fp, byte b)
 {
+    static std::array<uint64_t, 256> FP_TABLE = init_fp_table();
```

Comment:
will it create a table for each call?

Reply:
It should be called just once as it is a static variable. So one table is created.